spec/system/authenticate_user_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe "Authentication", type: :system, service: :claims do
```

Comment:
If this is scoped to the claims service, we might want to put it in `spec/system/claims`.

As it's pretty straight-forward, I think we could save School Placements some time by adding the same file for that side.